draft-packages/avatar/package.json

```diff
@@ -38,7 +38,7 @@
     "react-textfit": "^1.1.1"
   },
   "peerDependencies": {
-    "@kaizen/design-tokens": "^2.10.3 || ^3.0.0",
+    "@kaizen/design-tokens": "^2.10.3 || ^3.0.0 || ^6.1.0",
```

Comment:
I think this could be `^6.0.0` rather than `^6.1.0`?

Also might be worth adding `5.0.0` to these as well

Reply:
That was a speedy review 😄 

I will update them all to be `^6.0.0` and include `5.0.0`